android/src/main/java/com/evollu/react/fcm/FIRMessagingModule.java

```diff
@@ -54,12 +60,38 @@ public void requestPermissions(){
     }
 
     @ReactMethod
+    public void getInitFCMData(Promise promise) {
```

Comment:
when user clicks notification while app is open, intent for current activity will be changed to the intent of notification.
I still prefer to store initial data in the module when app starts so the value will remain unchanged.